utopia-remix/app/routes/projects.tsx

```diff
@@ -551,7 +546,11 @@ const ProjectCard = React.memo(
                     filter: project.deleted === true ? 'grayscale(1)' : undefined,
                   }}
                   title={collaborator.name}
-                  className={sprinkles({ boxShadow: 'shadow' })}
+                  className={sprinkles({
+                    boxShadow: 'shadow',
+                    color: 'white',
+                    background: 'primary',
```

Comment:
Should this be `backgroundColor`?

Reply:
yeah it should have been

Reply:
[`5999810` (#4936)](https://github.com/concrete-utopia/utopia/pull/4936/commits/59998107213f557c7040bbfb6352208ea8864607)